src/lib/types.ts

```diff
@@ -23,10 +23,12 @@ export enum Color {
 }
 
 export type UserColor = {
-	name: string;
-	primary: string;
-	secondary: string;
+	name: colorName;
+	primary: hexColorString;
+	secondary: hexColorString;
 };
+export type colorName = string;
+export type hexColorString = string;
 
```

Comment:
Husk pascalcase, ikke camelcase for typer i typescript. Altså ColorName og HexColorString